dot/core/service.go

```diff
@@ -350,15 +352,18 @@ func (s *Service) handleChainReorg(prev, curr common.Hash) error {
 				continue
 			}
 
-			externalExt := make(types.Extrinsic, 0, 1+len(ext))
-			externalExt = append(externalExt, byte(types.TxnExternal))
-			externalExt = append(externalExt, ext...)
-			txv, err := rt.ValidateTransaction(externalExt)
+			externalExt, err := s.buildExternalTransaction(rt, ext)
+			if err != nil {
+				logger.Debugf("building external transaction: %w", err)
```

Comment:
Shouldn't this be at least at the warning level? :thinking: 

Or does it happen a lot?

Reply:
You are right, but functions were now modified to return error here